RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
```

Comment:
Can you please delete the release file? I have to review another PR, and I think I'll create a release when the both PRs will be merged :)

Reply:
Yup, done.